.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+
+## Purpose
+
+Describe the purpose of this PR and the motivation if relevant to understanding. Include links to related issues, bugs or features. Include the sections below that apply.
+
+## Code changes:
+Provide a list of relevant code changes and the side effects they have on other code.
+
+## Requirements changes:
+Provide a list of any changes made to requirements or linting, e.g. changes to files requirements*.txt, constraints.txt, setup.py, pyproject.toml, pre-commit-config.yaml as well as a reason if not included in the Purpose section (e.g. incompatibility, updates, etc)
+
```

Comment:
Changes to linting requirements will already be reflected in code changes (because the code will need to be updated) and infrastructure changes (we are changing the testing infrastructure). If you like you could change the section heading to "Dependency changes".

```suggestion
## Requirements changes:

- Provide a list of any changes made to requirements, e.g. changes to files requirements*.txt, constraints.txt, setup.py, pyproject.toml, pre-commit-config.yaml and a reason if not included in the Purpose section (e.g. incompatibility, updates, etc)

```

Reply:
I meant more changes to the linting tools -- e.g. the configuration of the linting, but we don't have to say linting.